qiskit/opflow/gradients/hessian.py

```diff
@@ -244,7 +238,9 @@ def is_coeff_c(coeff, c):
 
             if operator.grad_combo_fn:
                 first_partial_combo_fn = operator.grad_combo_fn
-                if _HAS_JAX:
+                if _optionals.HAS_JAX:
```

Comment:
Same question about `require_now()` here as in gradient.py or decorating the `get_hessian()` method.